findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js

```diff
@@ -111,7 +111,7 @@ export default class HealthcareTeam extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
                     <Card
                         image={healthTeam}
-                        featuredTitleStyle={{ fontSize: 40 }}
+                        featuredTitleStyle={{ fontSize: 40, fontFamily: 'avenir-medium'}}

```

Comment:
The title font isn't overridden on Android:
![image](https://user-images.githubusercontent.com/22266870/79055159-5ffda200-7c10-11ea-98f0-30dbcf0722da.png)

Reply:
We're also missing fonts on the accordion:
![image](https://user-images.githubusercontent.com/22266870/79055341-e8c90d80-7c11-11ea-96bf-99993a2e98ed.png)